Makefile

```diff
@@ -119,16 +127,20 @@ clean:
 	@find ./ -name '*~' -exec rm -f {} \;
 
 .PHONY: version
-## dump package name into VERSION env variable and show
+## dump version into VERSION env variable and show
 version:
-	@export VERSION=$(grep __version__ setup.py | head -1 | cut -d \" -f2 | cut -d \' -f2)
-	@$(info VERSION is [${VERSION}])
+	@echo "VERSION: $(VERSION)"
+
+.PHONY: version-dev
+## dump version-dev into HASH_COMMIT env variable and show
+version-dev:
+	@sed -i 's/$(VERSION)/$(HASH_COMMIT)/g' setup.py
+	@echo "VERSION: $(HASH_COMMIT)"
```

Comment:
This command is not doing what you are saying here. `sed -i` will modify the `setup.py` file inplace (`-i`), replacing the content of the variable VERSION with the content of the variable HASH_COMMIT. As I was looking through your code, you actually provide the variable HASH_COMMIT as a parameter to this make command ([ref](here)), so the command itself works, but the description doesn't match what this make target is doing.

Also, wasn't the version supposed to be updated as `'s/$(VERSION)/$(VERSION)\.dev$(HASH_COMMIT)/g'` to be in the form of VERSION.devN that we talked on slack? In your case, this is replacing VERSION with HASH_COMMIT not appending to it.

Reply:
I will update the description, but the command is correct, the target name may be confusing, I will change it :smiley_cat:

Reply:
I was confused by the description and the env var. Now it looks good to me! :) The only thing that I'm afraid of is that the target `version` doesn't expect a parameter because it builds the `VERSION` variable within the makefile, while this new command `version-dev` requires you to pass `VERSION_DEV` as a parameter. So I would try to build it also inside the makefile, or, instead, require as parameter only the suffix that builds `VERSION_DEV`, so both commands seem similar. You could also implement only one target that does the same thing, where if the user provides a parameter HASH_COMMIT, it appends to the version, if not, then it just prints out the version.